From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Tue, 19 Jan 2021 22:18:55 +0000
Subject: [PATCH] data: Use the correct mode for the state dir

The code (in file_storage.c) expects mode 0700. However, systemd
defaults to 0755. This causes strange behavior.

https://bugs.archlinux.org/task/68932
---
 data/fprintd.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/fprintd.service.in b/data/fprintd.service.in
index 9ea7a2a..02bd13e 100644
--- a/data/fprintd.service.in
+++ b/data/fprintd.service.in
@@ -13,6 +13,7 @@ ProtectKernelTunables=true
 ProtectControlGroups=true
 # This always corresponds to /var/lib/fprint
 StateDirectory=fprint
+StateDirectoryMode=0700
 ProtectHome=true
 PrivateTmp=true
 
